### ISSUE TITLE
Node.js ERR_REQUIRE_ESM with v3

### ISSUE THREAD

**Issue by guillaumewuip** [GoogleChrome/web-vitals#256]
Hello!

Migrating to web-vitals v3, I have an issue with the lib used in a commonjs build run by Node.js.

The big picture for (simplified to remove the browser build part):
```
app <-- built to commonjs to be run in Node.js
  │
  +--- imports --- lib A <-- built to commonjs 
                    |
                    +----  imports --- web-vitals v3
```

When I run the app in Node.js, I get this error

```
Error [ERR_REQUIRE_ESM]: require() of ES Module .../libA/dist/commonjs/index.js not supported.
web-vitals.umd.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename web-vitals.umd.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in .../node_modules/web-vitals/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

Removing the `type: module` [from web-vitals `package.json`](https://github.com/GoogleChrome/web-vitals/blob/ab67c29403cca356bc0fc385780cc43011f5fb76/package.json#L5) indeed resolves the issue.

**Comment by tizmagik** [comment]
Thanks for creating the issue, I just hit this as well.

**Comment by philipwalton** [comment]
Thanks for reporting. I believe this should be fixed in [`v3.0.1`](https://github.com/GoogleChrome/web-vitals/blob/main/CHANGELOG.md#v301-2022-08-31), let me know if you're still experiencing issues.

**Comment by brmendez** [comment]
> Thanks for reporting. I believe this should be fixed in [`v3.0.1`](https://github.com/GoogleChrome/web-vitals/blob/main/CHANGELOG.md#v301-2022-08-31), let me know if you're still experiencing issues.

@philipwalton thanks for the reply. I was experiencing the above issues as well. I have upgraded to `v3.0.1` and am now getting this error:
```
ReferenceError: addEventListener is not defined
      at l (node_modules/web-vitals/dist/web-vitals.umd.cjs:1:1249)
      at h (node_modules/web-vitals/dist/web-vitals.umd.cjs:1:1649)
      at g (node_modules/web-vitals/dist/web-vitals.umd.cjs:1:1725)
      at y (node_modules/web-vitals/dist/web-vitals.umd.cjs:1:1865)
```
